include/internal/quic_param.h

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
This file does not seem to belong to this PR as these constants do not look relevant for the wire format support.

Reply:
It's intended for use with the transport parameter encoding functions, but you're correct that we could do without this for now.